http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -102,6 +104,64 @@ public void abstractResourceWithPath() {
                 .body(is("Hello from Quarkus REST"));
     }
 
+    @Test
+    @DisplayName("Test Quarkus REST using CDI getting the sub-resource")
+    void cdiSubResourceGetRequest() {
+        getApp().given()
+                .when()
+                .get(GREETING_ENDPOINT + "/cdi-sub-resource")
+                .then()
+                .statusCode(SC_OK)
+                .body(is("Greeting from sub-resource using GET"));
+    }
+
+    @Test
+    @DisplayName("Test Quarkus REST using CDI getting the sub-resource")
+    void cdiSubResourcePostRequest() {
+        getApp().given()
+                .when()
+                .post(GREETING_ENDPOINT + "/cdi-sub-resource")
+                .then()
+                .statusCode(SC_OK)
+                .body(is("Greeting from sub-resource using POST"));
+    }
+
+    @Test
+    @DisplayName("Test Quarkus REST interface resource with @Path")
+    void interfaceResourceWithPath() {
+        getApp().given()
+                .when()
+                .get(GREETING_ENDPOINT + "/interface-greeting")
+                .then()
+                .statusCode(SC_OK)
+                .body(is("Greeting from interface"));
+    }
+
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/issues/43422")
+    @DisplayName("Test Quarkus REST abstract resource with @Path using OPTION and HEAD request")
+    @DisabledOnNative(reason = "https://github.com/quarkusio/quarkus/issues/42976")
+    void optionAndHeadRequestUsingAbstractResourceWithPath() {
+        testOptionRequest(GREETING_ENDPOINT);
+        testHeadRequest(GREETING_ENDPOINT);
+    }
+
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/issues/43422")
+    @DisplayName("Test Quarkus REST interface resource with @Path using OPTION and HEAD request")
+    void optionAndHeadRequestUsingInterfaceResourceWithPath() {
+        testOptionRequest(GREETING_ENDPOINT + "/interface-greeting");
+        testHeadRequest(GREETING_ENDPOINT + "/interface-greeting");
+    }
+
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/issues/43422")
+    @DisplayName("Test Quarkus REST using CDI getting the sub-resource for OPTION and HEAD request")
+    void cdiSubResourceOptionAndHeadRequest() {
```

Comment:
This is only method that fails with 3.15.1. Why are other tests `optionAndHeadRequestUsingInterfaceResourceWithPath` and `optionAndHeadRequestUsingAbstractResourceWithPath` tagged with `https://github.com/quarkusio/quarkus/issues/43422` if they are passing with 3.15.1?

Reply:
I added the tag there as this issue was main motivation to create that coverage. If you things tha tag shouldn't be there. I can create PR for its removal.

Reply:
Clear answer :-D : 

I don't really care, keep it as it. I just thought that if you tag something with an issue, it means that issue is tested by this. It can still be true that changed code is not failing but testing changes because it can test code that we didn't have covered previously. What I do when I verify bug tickets is that I look for tests tagged with that issue. Now I'll keep in mind that failing test tagged with an issue may not mean that issue is not fixed. It probably won't happen anyway.

Reply:
Also, why are other 3 tests added in this PR without this tag, is that because they were not added because of this issue or you just forgot?

Reply:
Goodcatch @michalvavrik. @jedla97 can you prepare a followup PR with backport label?

Reply:
> Also, why are other 3 tests added in this PR without this tag, is that because they were not added because of this issue or you just forgot?

There are more like to ensure that endpoint work. The quarkus fix was to add `HEAD` and `OPTION`. At least this is my logic behind adding the tag here.

> @jedla97 can you prepare a followup PR with backport label?

@rsvoboda with adding or removing the tags?

Reply:
I think this kinda depends on how other QE members use `@Tag`, I always thought that it signals that some test tests the issue/ticket. But maybe that was just me.

Reply:
`@Tag` usage ... there is no rule really, some members started to use it, but we didn't have any discussion and  conclusion on its usage.

I would personally use `@Tag` only on tests testing the bug. I wouldn't tag the other related tests.